### PR TITLE
Fix Byron CDDL link

### PR DESCRIPTION
**Pull request by SebastienGllmt**
Byron CDDL link was accidentally instead linking to the spec PDF

### CODE REVIEW

**Comment by JaredCorduan** [comment]
This was not an accident, the PDF of Byron wire spec is actually very nice. At the end of it, in Appendix A, you'll see the exact same information as the file you linked, `eras/byron/cddl-spec/byron.cddl`.

Perhaps we could link both, if there is some reason (consistency maybe?) to include `byron.cddl`.

**Comment by JaredCorduan** [comment]
@SebastienGllmt does my response make sense?

**Comment by SebastienGllmt** [comment]
yeah I guess it makes sense, but maybe linking both is a good solution for people who are looking for the cddl file specifically

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ following links:
 
 Era | Design Documents | Formal Specification | CDDL
 ----|------------------|----------------------|-----
-Byron | | [Chain Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronChainSpec/latest/download-by-type/doc-pdf/blockchain-spec "Specification of the Blockchain Layer"), [Ledger Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec "A Formal Specification of the Cardano Ledger") | [CDDL](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/blocksCDDLSpec/latest/download-by-type/doc-pdf/binary)
+Byron | | [Chain Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronChainSpec/latest/download-by-type/doc-pdf/blockchain-spec "Specification of the Blockchain Layer"), [Ledger Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec "A Formal Specification of the Cardano Ledger") | [CDDL](https://github.com/input-output-hk/cardano-ledger/tree/master/eras/byron/cddl-spec/byron.cddl)
```
Comment (JaredCorduan) [inline README.md]:
```suggestion
Byron | | [Chain Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronChainSpec/latest/download-by-type/doc-pdf/blockchain-spec "Specification of the Blockchain Layer"), [Ledger Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/byronLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec "A Formal Specification of the Cardano Ledger") | [CDDL](https://github.com/input-output-hk/cardano-ledger/tree/master/eras/byron/cddl-spec/byron.cddl) [PDF](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/blocksCDDLSpec/latest/download-by-type/doc-pdf/binary)
```

Comment (JaredCorduan) [reply README.md]: @SebastienGllmt ?

**Review by JaredCorduan — APPROVED** [review]

**Pull request merged**